src/styles/_reset.scss

```diff
@@ -47,6 +47,10 @@ td {
     padding: 0;
 }
 
+figure {
```

Comment:
This may lead to some unexpected styling in our consuming apps, as this stylesheet is widely used. Can we move the override to just where it's needed?